nix/nixos-module.nix

```diff
@@ -14,12 +14,12 @@ with lib;
       '';
     };
 
-    configfile = mkOption {
-      type = types.path;
-      default = "";
-      example = "my-config.kbd";
+    configfiles = mkOption {
+      type = types.listOf types.path;
+      default = [];
+      example = "[ my-config.kbd ]";
```

Comment:
Aside from the pluralization of the option, this also takes a list of config paths. This was the breaking change i was referring to since someone might just pull in this repo from master and load this into their config via overlay.